vault/logical_system.go

```diff
@@ -1586,7 +1606,31 @@ func (b *SystemBackend) handleTuneWriteCommon(
 		}
 	}
 
-	return nil, nil
+	var resp *logical.Response
```

Comment:
This doesn't seem to be used anywhere, should we just return `nil, nil` below?

Reply:
Ah, it was used for a warning before but that code is now removed, will fix it.